wfl-slack-app/src/wfl-api.js

```diff
@@ -0,0 +1,98 @@
+var request = require('request');
+
+function postFoodOrder(restaurantId, callback) {
+    let options = {
+        url: process.env.DB_URI + '/food-order',
+        form: {
+            RestaurantId: restaurantId
+        }
+    }
+
+    request.post(options, function(err, res, body) {
+        let json = JSON.parse(body);
+        if (json) {
+            callback(json);
+        }
+    });
+}
+
+function getAllRestaurants(callback) {
+    let options = {
+        url: process.env.DB_URI + '/restaurant'
+    }
+    request.get(options, function(err, res, body) {
+        callback(JSON.parse(body));
+    });
+}
+
+function getFoodOrderById(id, callback) {
```

Comment:
Why did we choose to use both fetch and request for REST APIs?

Reply:
I mentioned this in the other comment about fetch, but I did this just because I wanted to be able to reuse the code from the wfh app when calling to the slack api. I could convert the slack-api.js code to use request instead if you think that would make for better uniformity across classes.